compiler.c

```diff
@@ -1463,6 +1533,8 @@ static int lbp(enum token_type op)
 
 	case TOK_LEFT_PAREN:
 	case TOK_LEFT_BRACKET:
+	case TOK_COLON:
+	case TOK_LEFT_BRACE:
```

Comment:
Test `struct a {b} a { b = 1 }:b;` and `getstruct() { a = 2 }` and `a { b = a { b = 3 } }:b:b`